packages/Tabs/src/components/Tab/Tab.js

```diff
@@ -0,0 +1,76 @@
+import React from "react";
+import PropTypes from "prop-types";
+import RawButton from "@paprika/raw-button";
+import TabsContext from "../../TabsContext";
+import { tabStyles } from "./Tab.styles";
+
+const propTypes = {
+  children: PropTypes.node,
+  className: PropTypes.string,
+  isSelected: PropTypes.bool,
+  isDisabled: PropTypes.bool,
+  href: PropTypes.string,
+  onClick: PropTypes.func,
+  onKeyDownArrows: PropTypes.func,
+};
+
+const defaultProps = {
+  children: null,
+  className: null,
+  isSelected: false,
+  isDisabled: false,
+  href: null,
+  onClick: () => {},
+  onKeyDownArrows: () => {},
+};
+
+export default function Tab(props) {
+  const context = React.useContext(TabsContext);
+
+  const { className, isSelected, children, href, onClick, onKeyDownArrows, ...moreProps } = props;
+
+  const handleKeyDown = (event, index) => {
+    onKeyDownArrows(event, index);
+  };
+
+  const isDisabled = context.isDisabled || props.isDisabled;
+  const handleClick = isDisabled ? () => {} : onClick;
+  const tabIndex = isSelected ? 0 : -1;
+
+  if (href) {
+    return (
+      <a
+        css={tabStyles}
```

Comment:
Since the focus ring styling is coming from `<RawButton>`, when the tab is a link, the focus ring styling is inconsistent (default browser styling).